ui.go

```diff
@@ -116,10 +133,11 @@ func (u *ui) loadUI() {
 		widget.NewFormItem("Password", u.pass),
 		widget.NewFormItem("Session", u.session))
 	f := widget.NewForm(formItems...)
-	f.SubmitText = "Log In"
-	f.CancelText = "Shutdown"
-	f.OnCancel = u.askShutdown
-	f.OnSubmit = u.doLogin
+	login := widget.NewButtonWithIcon("Log In", theme.LoginIcon(), u.doLogin)
+	login.Importance = widget.HighImportance
+	buttons := container.NewGridWithColumns(2,
+		widget.NewButtonWithIcon("Shutdown...", theme.NewThemedResource(resourcePowerSvg), u.askShutdown),
```

Comment:
I don't know so if this is something that is generally accepted for uis, but I think this would look better without the three dots at the end.

Reply:
It is used in menus to indicate that another dialog will come up instead of performing the action. 
But you're right it doesn't look good.